apps/rsautl.c

```diff
@@ -31,7 +31,7 @@ NON_EMPTY_TRANSLATION_UNIT
 typedef enum OPTION_choice {
     OPT_ERR = -1, OPT_EOF = 0, OPT_HELP,
     OPT_ENGINE, OPT_IN, OPT_OUT, OPT_ASN1PARSE, OPT_HEXDUMP,
-    OPT_RAW, OPT_OAEP, OPT_SSL, OPT_PKCS, OPT_X931,
+    RSA_OPT_RAW, OPT_OAEP, OPT_SSL, OPT_PKCS, OPT_X931,
```

Comment:
what is the reason for this change?

Reply:
there is a naming conflict with vxworks...

Reply:
how unfortunate, but you cannot just change one prefix.

Reply:
Maybe OSSL_OPT_xxx or OPTION_xxx or something.

Reply:
ok, will change the otheres as well...
OSSL_OPT_... sounds good to me

Reply:
another less intrusive possibility say OPT_NO_PADDING?

Reply:
The point is my grep tells me OPT_xxx is all over everywhere..

Reply:
OPT_RAW_RSA is what it must be.

Reply:
Hi Yoda.